packages/paste-website/src/pages/utilities/grid/index.mdx

```diff
@@ -0,0 +1,567 @@
+
+<LivePreview scope={{Grid, Column, Box, Card, Heading, Paragraph}} language="jsx">
+  {`<Grid vertical>
+  <Column>
+    <Grid gutter="space30">
+      <Column>
+        <Card padding="space70">
+          <Heading as="h4" variant="heading40">Getting Started</Heading>
+          <Paragraph>To get started with Phone Numbers, check out our Getting Started Guide.</Paragraph>
+        </Card>
+      </Column>
+      <Column>
+        <Card padding="space70">
+          <Heading as="h4" variant="heading40">About Web API</Heading>
+          <Paragraph>The fastest, most flexible way to send email using languages like Node.js, Ruby, C#, and more.</Paragraph>
+        </Card>
+      </Column>
+    </Grid>
+  </Column>
+  <Column>
+    <Box marginTop="space60" marginBottom="space60">
+      <Card padding="space70">
+        <Heading as="h4" variant="heading40">SMTP Relay</Heading>
+        <Paragraph>The easiest way to send email. It only requires modifying your application's SMTP configuration.</Paragraph>
+      </Card>
+    </Box>
+  </Column>
+  <Column>
+    <Grid gutter="space30">
+      <Column>
+        <Card padding="space70">
+          <Heading as="h4" variant="heading40">About Phone Numbers</Heading>
+          <Paragraph>Programmable phone numbers are a core part of Twilio’s platform, enabling you to receive SMS, MMS, and phone calls.</Paragraph>
+        </Card>
+      </Column>
+      <Column>
+        <Card padding="space70">
+          <Heading as="h4" variant="heading40">Messaging Services</Heading>
+          <Paragraph>Messaging Services allow you to organize your messages and enable specific features for groups of messages.</Paragraph>
+        </Card>
+      </Column>
+    </Grid>
+  </Column>
+</Grid>`}
+</LivePreview>
+
+### Responsive columns
+
+Grid column span and offset props can be changed based on the theme breakpoints. Both span and offsets can accept responsive values, for example `span={[6, 8, 2] }`.
+This would set a columns span to be 6 columns wide on mobile, 8 columns wide on tablet, and 2 columns wide on desktop. The offset prop works in a similar fashion.
+For a visual example, [see the Storybook example](https://twilio-labs.github.io/paste/?path=/story/utilities-grid--grid-2-column-responsive) and resize the browser.
+
+
+## Composition Notes
+
+Grid can be used to compose various types of layouts ranging from whole application pages and views down to individual component layouts.
+
+<Callout variant="secondary">
+  <CalloutText>For spacing content in a layout wider than 1440 pixels, we recommend using a 24-pixel gutter. Use $space-40 to get a 24-pixel gutter.</CalloutText>
+</Callout>
+
+<Callout variant="secondary">
+  <CalloutText>For spacing content in a layout narrower than 1440 pixels, we recommend using a 16-pixel gutter. Use $space-30 to get a 16-pixel gutter.</CalloutText>
+</Callout>
```

Comment:
These could be combined into a single Callout in the default variant, and with a heading like "How to size gutters" or "Gutter sizing"